### ISSUE TITLE
Tracker of etingof/snmpsim issues

### ISSUE THREAD

**Issue by lextm** [lextudio/pysnmp#25]
| Issue | Status | Comments |
| ----- | ------- | ---------- |
| inexio/snmpsim/issues/17 | TODO |  |
| inexio/snmpsim/issues/16 | TODO | feature request |
| inexio/snmpsim/issues/15 | TODO | py2exe related.  |
| inexio/snmpsim/issues/10 | TODO | feature request |
| etingof/snmpsim/issues/172 | TODO |  |
| etingof/snmpsim/issues/170 | TODO |  |
| etingof/snmpsim/issues/169 | Pending on user | Explanation was provided. |
| etingof/snmpsim/issues/167 | TODO | Seem to be using broken package. |
| etingof/snmpsim/issues/166 | TODO | Duplicate to 167. |
| etingof/snmpsim/issues/165 | TODO |  |
| etingof/snmpsim/issues/164 | TODO | Related to runtime user account permissions. |
| etingof/snmpsim/issues/163 | Pending on user | Resolved by new patches. |
| etingof/snmpsim/issues/162 | TODO | About SQL. |
| etingof/snmpsim/issues/161 | TODO | Feature request. |
| etingof/snmpsim/issues/156 | TODO | Different opinion on design. |
| etingof/snmpsim/issues/152 | TODO |  |
| etingof/snmpsim/issues/143 | TODO |  |
| etingof/snmpsim/issues/136 | TODO |  |
| etingof/snmpsim/issues/135 | TODO |  |
| etingof/snmpsim/issues/134 | TODO | A nice-to-have path fix.  |
| etingof/snmpsim/issues/131 | TODO |  |
| etingof/snmpsim/issues/123 | TODO | HP specific  |
| etingof/snmpsim/issues/121 | TODO |  feature request |
| etingof/snmpsim/issues/120 | TODO |  feature request   |
| etingof/snmpsim/issues/119 | TODO |  feature request   |
| etingof/snmpsim/issues/118 | TODO |  feature request   |
| etingof/snmpsim/issues/117 | TODO |  feature request   |
| etingof/snmpsim/issues/116 | TODO |  feature request   |
| etingof/snmpsim/issues/113 | TODO | feature request on sapwalk. |
| etingof/snmpsim/issues/109 | TODO | doc issue. |
| etingof/snmpsim/issues/107 | TODO |   feature request  |
| etingof/snmpsim/issues/106 | TODO |   feature request |
| etingof/snmpsim/issues/96 | TODO | Windows specific |
| etingof/snmpsim/issues/95 | TODO |  |
| etingof/snmpsim/issues/94 | TODO | walk/datafile related. |
| etingof/snmpsim/issues/91 | TODO |  |
| etingof/snmpsim/issues/83 | TODO |  |
| etingof/snmpsim/issues/79 | TODO |  |
| etingof/snmpsim/issues/78 | TODO | feature request |
| etingof/snmpsim/issues/75 | TODO |  |
| etingof/snmpsim/issues/68 | TODO |  |
| etingof/snmpsim/issues/66 | TODO | constraints related. |
| etingof/snmpsim/issues/61 | TODO | root related. |
| etingof/snmpsim/issues/49 | TODO | feature request |
| etingof/snmpsim/issues/41 | TODO |  |
| etingof/snmpsim/issues/38 | TODO | feature request |
| etingof/snmpsim/issues/37 | TODO |  |
| etingof/snmpsim/issues/34 | TODO | feature request |
| etingof/snmpsim/issues/11 | TODO | pypy support |
| etingof/snmpsim/issues/7 | TODO | feature request |
